docs/site/migration/auth/passport.md

```diff
@@ -10,8 +10,378 @@ permalink: /doc/en/lb4/migration-auth-passport.html
 Missing instructions for your LoopBack 3 use case? Please report a [Migration docs issue](https://github.com/strongloop/loopback-next/issues/new?labels=question,Migration,Docs&template=Migration_docs.md) on GitHub to let us know.
 " %}
 
-{% include note.html content="
-This is a placeholder page, the task of adding content is tracked by the
-following GitHub issue:
-[loopback-next#3958](https://github.com/strongloop/loopback-next/issues/3958)
-" %}
+# Migrating Passport Strategies
+
+This page guides on migrating lb3 apps that used
+[Passport Strategies](http://www.passportjs.org/) for authentication usecases.
+Before following this guide, please know more about the
+[@loopback/authentication-passport](https://loopback.io/doc/en/lb4/Authentication-passport.html)
+package.
+
+## Differences between LoopBack 3 and LoopBack 4
+
+In LoopBack 3, routes can be configured explicitly as authentication providers
+using express style passport strategies middleware. Also the
+[lb3 passport component](https://github.com/strongloop/loopback-component-passport)
```

Comment:
Perhaps it's more consistent to use `LB3` instead of `lb3` throughout the document? Similarly for `LB4`.

Reply:
agreed